src/hooks/useBookSearch.jsx

```diff
@@ -37,15 +42,13 @@ export default function useBookSearch(query, orderBy, pageNumber, setAreResultsL
           setBooks((prevBooks) => {
             if (pageNumber === 1) {
               setQueryHistory((q) => {
-                //Prevent duplicate queries being added to query history
```

Comment:
This comment is also useful

Reply:
I reinstated some of the useful comments in App.scss & useBookSearch.jsx, please double-check & confirm :)